lib/ngsiUtils.js

```diff
@@ -22,10 +22,27 @@
  */
 'use strict';
 
-var request = require('request');
+var request = require('request'),
+    commons = require('./commons');
 
 /**
- * Updates the information in a Context Broker entity.
+ * It checks if the configuration file states the use of NGSIv2
+ *
+ * @param      {Object} newConfig            Configuration of the Context Server
+ * @return     {boolean}  Result of the checking
+ */
+function checkNgsi2(newConfig) {
```

Comment:
I think to remember the same check at IOTA Library... Do we need double-check (one in the library, one in the agent code)? Could duplicated code be avoided?

Reply:
Yes, a similar function exists in https://github.com/dcalvoalonso/iotagent-node-lib/blob/task/activeAttsNgsi2/lib/commonConfig.js#L253.

To use it in the IOTAs, we would have to export it in https://github.com/dcalvoalonso/iotagent-node-lib/blob/task/activeAttsNgsi2/lib/fiware-iotagent-lib.js#242 using something like:

```
exports.configModule = config;
```

It this OK for you?

Reply:
Yes, I think that's the way. From my point of view, the library is the "owner" of these variables in config.js that are general, as the v2 configuration. Thus, better to ask the library from the agent that check the config file directly.

Reply:
It makes sense to me, let's going to do so!

Reply:
Fixed in 5023f484d0afba73bfa4ba7a6ef3c17c43187eb7